openstack/neutron/templates/statefulset-network-agent.yaml

```diff
@@ -166,11 +162,7 @@ spec:
           imagePullPolicy: IfNotPresent
           command: ["dumb-init", "neutron-l3-agent", "--config-file", "/etc/neutron/neutron.conf", "--config-dir", "/etc/neutron/secrets", "--config-file", "/etc/neutron/l3-agent.ini", "--config-file", "/etc/neutron/plugins/ml2/ml2-conf.ini", "--config-file", "/etc/neutron/linux-bridge.ini", "--config-file", "/etc/neutron/az-{{ $az }}.conf"]
           env:
-            - name: SENTRY_DSN
-              valueFrom:
-                secretKeyRef:
-                  name: sentry
-                  key: neutron.DSN.python
+            {{- include "utils.sentry_config" . | nindent 12 }}
```

Comment:
The `utils.trust_bundle.env` right below uses `$` not `.` - which one is correct?